ogusa/tests/test_SS.py

```diff
@@ -933,6 +935,121 @@ def test_euler_equation_solver(dask_client):
     assert(np.allclose(np.array(test_list), np.array(expected_list)))
 
 
+@pytest.mark.local
```

Comment:
Why is the a local test? Isn't this a quick test to run?

Reply:
@jdebacker . `UBI>0` is not a baseline use case, and I didn't want to add time to the already close-to-the-limit CI test suite. So my approach was to separate all `UBI>0` tests to be run locally and not in the CI.